packaging/windows/GetWix.ps1

```diff
@@ -0,0 +1,28 @@
+﻿Param(
+    [string]$WixVersion="3.10.2",
```

Comment:
Why default this version when the caller always passes it in?  The version is duplicated here and in the props file.